src/pages/ImportAnalysis.js

```diff
@@ -0,0 +1,419 @@
+import _ from 'lodash/fp';
```

Comment:
is there any reason not to introduce this new file as a typescript file? Seems pretty unsafe to introduce a javascript file when a lot of the underlying imports would be safer consumed with type checkcing